Sources/PackageDescription/PackageDependency.swift

```diff
@@ -64,6 +64,23 @@ extension Package.Dependency {
     ) -> Package.Dependency {
         return .init(name: name, url: url, requirement: .upToNextMajor(from: version))
     }
+    
+    /// Adds a remote package dependency given a branch requirement.
+    ///
+    ///    .package(url: "https://example.com/example-package.git", branch: "main"),
+    ///
+    /// - Parameters:
+    ///     - name: The name of the package, or nil to deduce it from the URL.
+    ///     - url: The valid Git URL of the package.
+    ///     - branch: A dependency requirement. See static methods on `Package.Dependency.Requirement` for available options.
+    @available(_PackageDescription, introduced: 5.3)
```

Comment:
This should be 999.0 until the version number of the next release is announced.

Reply:
Ah okay, I wasn't sure so I just did the current version, I'll take care of that and the other parts in the comment